configure

```diff
@@ -728,7 +728,7 @@ def configure_library(lib, output):
     # libpath needs to be provided ahead libraries
     if pkg_libpath:
       output['libraries'] += (
-          filter(None, map(str.strip, pkg_cflags.split('-L'))))
+          map('-L{0}'.format, filter(None, map(str.strip, pkg_libpath.split('-L')))))
```

Comment:
@walkerlee isn't this code just removing `'-L'` from each library and then adding it back at the end?

Would the appropriate course of action not be just to split `pkg_libpath` on whitespace?